### ISSUE TITLE
Register the feedstock with Azure

### ISSUE THREAD

**Issue by jdblischak** [TileDB-Inc/m2w64-htslib-feedstock#1]
@ihnorton Here are the steps you (or someone else with access to the TileDB GitHub and Azure accounts) will need to follow to register this new feedstock with Azure:

1. Go to https://dev.azure.com/TileDB-Inc/_usersSettings/tokens to create a new Azure token. I haven't figured out the exact minimum set of scopes required, but I'd recommend setting at least Build (Read & Execute), Code (Read), and Service Connections (Read & query) (I know the latter is required, because this is how Azure communicates with GitHub)
1. Copy-paste the token to your local machine in a file named `~/.conda-smithy/azure.token`
1. Install conda-smithy locally
    ```sh
    mamba create -n smithy -c conda-forge conda-smithy
    conda activate smithy
    ```
1. Clone the repo `git clone https://github.com/TileDB-Inc/m2w64-htslib-feedstock.git`
1. Register the feedstock with Azure
    ```sh
    export AZURE_ORG_OR_USER=TileDB-Inc
    export AZURE_PROJECT_NAME=CI
    conda smithy register-ci \
      --organization TileDB-Inc \
      --feedstock_directory ./m2w64-htslib-feedstock \
      --without-travis \
      --without-circle \
      --without-appveyor \
      --without-drone \
      --without-webservice \
      --without-anaconda-token
    ```
1. Go to Azure and confirm there is now a pipeline named "m2w64-htslib-feedstock" https://dev.azure.com/TileDB-Inc/CI/_build?view=folders

**Comment by ihnorton** [comment]
Done. I had to enable all permissions temporarily to get it to go through. I cut the permissions back down to what you listed plus "task groups", but let me know if it looks like more are required. I can't find a list of what is required by conda-forge...

**Comment by ihnorton** [comment]
See https://dev.azure.com/TileDB-Inc/CI/_build?definitionId=46

**Comment by jdblischak** [comment]
> I cut the permissions back down to what you listed plus "task groups", but let me know if it looks like more are required. I can't find a list of what is required by conda-forge...

Right, it's frustrating. It's not documented, so it's been completely trial and error for me. My original Azure token I created for my nightlies prototype expired at the beginning of this year, so I had to do it a second time this week.

After I merged the PR, a build was triggered, and status icons connected to the commit in GitHub, so that should be all we need for now.

Thanks!
